components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -185,6 +186,12 @@ export default class CreateWorkspace extends React.Component<CreateWorkspaceProp
                     phase = StartPhase.Stopped;
                     statusMessage = <LimitReachedOutOfHours />;
                     break;
+                case ErrorCodes.INVALID_COST_CENTER:
+                    // HACK: Hide the error (behind the modal)
+                    error = undefined;
+                    phase = StartPhase.Stopped;
+                    statusMessage = <SelectCostCenterModal onSelected={() => this.createWorkspace()} />;
```

Comment:
`onSelected => createWorkspace()` doesn't seem to invalidate error state, I think this might delay the visual effect, e.g. the transition from the modal to next state. maybe we should clear the error state therein, or try to reload when selection was done.